content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -2793,6 +2800,124 @@ Bans the customer for a specific period of time. It immediately disconnects all
 <Section>
 <Text>
 
+### Follow Customer
+
+Marks a customer as followed. As a result, the requester (an agent) will receive the info about all the changes related to that customer via pushes. Once the customer leaves the website or is [unfollowed](#unfollow-customer), the agent will no longer receive that information. 
+
+Following a customer causes the agent to receive customer-related pushes as long as the followed customer is available. The followed customer may be transferred to a group the agent doesn't have access to, which makes the customer unavailable to the agent. At that moment, the agent stops receiving pushes related to that customer. When the customer is transferred back and becomes available again, the pushes resume. 
+
+Agents don't need to follow the customers they're chatting with in order to receive related pushes. Agents will be receiving pushes related to those customers as long as the chats last. 
+
+
+#### Specifics
+
+|                        |                                                 |
+| ---------------------- | ----------------------------------------------- |
+| **Action**             | `follow_customer`                               |
+| **Required scopes**    | `customers:ro`                                  |
+| **Web API equivalent** | [`follow_customer`](../#follow-customer)        |
+| **Push message**       | [`incoming_customer`](#incoming_customer)**\*** |
+
+**\*)**
+It won't be sent when the requester already follows the customer or is chatting with that customer.
+
+#### Request
+
+| Parameter  | Required | Data type |     |
+| ---------- | -------- | --------- | --- |
+| `id`       | Yes      | `string`  |     |
+
+</Text>
+<Code>
+<CodeSample path={'REQUEST'}>
+
+```json
+{
+  "action": "follow_customer",
+  "payload": {
+    "id": "b7eff798-f8df-4364-8059-649c35c9ed0c"
+  }
+}
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+  "request_id": "<request_id>", // optional
+  "action": "follow_customer",
+  "type": "response",
+  "success": true,
+  "payload": {
+  // no response payload
+  }
+}
+```
+
+</CodeResponse>
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+### Unfollow Customer
+
+Removes the agent from the list of customer's followers. Calling this method on a customer the agent's chatting with will result in success, however, the agent will still receive pushes about the customer's data updates. The unfollowing will take effect once the chat ends.
```

Comment:
could you copy-paste it to Web reference as well?

Reply:
done